internal/cli/app/list_command.go

```diff
@@ -0,0 +1,67 @@
+package app
```

Comment:
I put this file flat with the app command.go file temporarily. I felt it was easier to do it this way and won't matter in the long run once this is refactored with the subcommand pattern discussed earlier

Reply:
this is fine here for now (though, the `list_command` and its corresponding test file should have names that match each other).

ultimately this file would live in a directory at `internal/cli/app/list/command.go`, but again, this is totally fine as is for now.  i'll be handling the major refactor stuff in a follow-up PR to this one